pkg/view/component/timestamp.go

```diff
@@ -27,12 +27,21 @@ type TimestampConfig struct {
 
 // NewTimestamp creates a timestamp component
 func NewTimestamp(t time.Time) *Timestamp {
-	return &Timestamp{
-		Base: newBase(TypeTimestamp, nil),
-		Config: TimestampConfig{
-			Timestamp: t.Unix(),
-		},
+
+	actualTime, err := time.Parse(time.RFC1123 ,t.String())
```

Comment:
Parsing the time would help us sanitize and see if the format is correct. Any other approaches are welcome

Reply:
Check https://github.com/vmware-tanzu/octant/blob/master/internal/printer/event.go#L65-L66 and `PrintEvents` for nil before creating the timestamps, otherwise use a text component to show they are missing.

Reply:
Done :+1: